### PR TITLE
Project documentation

### PR DESCRIPTION
**Pull request by emma-rachiteanu**
Pull Request Overview

This pull request adds the Project Documentation for Robo Tic Tac Toe.

TODO or Help Wanted

This pull request is still a draft. Further modifications of the documentation will be uploaded here.

Author

Signed-off-by: Răchițeanu Emma <rachiteanuemma@gmail.com>

### CODE REVIEW

File: docs/project/2024/emma.rachiteanu/img/front view robotic arm.jpg
Comment (alexandruradovici) [inline docs/project/2024/emma.rachiteanu/img/front view robotic arm.jpg:1]: Please use file names without spaces.

File: docs/project/2024/emma.rachiteanu/index.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+### Week 6 - 12 May
+
+### Week 7 - 19 May
+
+### Week 20 - 26 May
+
+## Hardware
+
+1. *]The Arm
```
Comment (alexandruradovici) [inline docs/project/2024/emma.rachiteanu/index.md:41]: I think this is a typo.

File: docs/project/2024/emma.rachiteanu/index.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+<!-- 
+| [Device](link://to/device) | This is used ... | [price](link://to/store) |
+-->
+
+| Device | Usage | Price |
+|--------|--------|-------|
+| [Rapspberry Pi Pico W](https://www.raspberrypi.com/documentation/microcontrollers/raspberry-pi-pico.html) | The microcontroller | [35 RON](https://www.optimusdigital.ro/en/raspberry-pi-boards/12394-raspberry-pi-pico-w.html) |
+| [Motor Servo (probably) SG90 9g](https://datasheetspdf.com/datasheet-pdf/791970/SG90.html) | The servomotors | [15 RON](https://www.emag.ro/servomotor-sg90-180-de-grade-ai0156-s297/pd/D33V1GMBM/) |
+| [ Display LCD TFT ](link://to/device) | Display of the game | [64 RON](hhttps://ardushop.ro/en/home/1029-display-lcd-tft-32-320x240-with-touch-for-stm32.html?search_query=display+3%22&results=87) |
```
Comment (alexandruradovici) [inline docs/project/2024/emma.rachiteanu/index.md:61]: The link is not valid.

File: docs/project/2024/emma.rachiteanu/index.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+|---------|-------------|-------|
+| [st7789](https://github.com/almindor/st7789) | Display driver for ST7789 | Used for the display for the Pico Explorer Base |
+| [embedded-graphics](https://github.com/embedded-graphics/embedded-graphics) | 2D graphics library | Used for drawing to the display |
+
+## Links
+
+<!-- Add a few links that inspired you and that you think you will use for your project -->
+
+1. [link](https://docs.sunfounder.com/projects/ultimate-sensor-kit/en/latest/components_basic/27-component_servo.html)
+2. [link](https://github.com/Makerfabs/PICO_Merchanical_Hand_Driver)
```
Comment (alexandruradovici) [inline docs/project/2024/emma.rachiteanu/index.md:78]: Please give titles to your links.

File: docs/project/2024/emma.rachiteanu/index.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+
+
+
+## Software
+
+| Library | Description | Usage |
+|---------|-------------|-------|
+| [st7789](https://github.com/almindor/st7789) | Display driver for ST7789 | Used for the display for the Pico Explorer Base |
+| [embedded-graphics](https://github.com/embedded-graphics/embedded-graphics) | 2D graphics library | Used for drawing to the display |
```
Comment (alexandruradovici) [inline docs/project/2024/emma.rachiteanu/index.md:71]: Do you need these libraries?

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/2024/emma.rachiteanu/index.md (hunk 4 above)
Comment (emma-rachiteanu) [reply docs/project/2024/emma.rachiteanu/index.md:71]: I forgot to change them from the basic template, thank you for pointing this out! I added useful libraries.

File: docs/project/2024/emma.rachiteanu/img/front view robotic arm.jpg
Comment (alexandruradovici) [inline docs/project/2024/emma.rachiteanu/img/front view robotic arm.jpg:1]: Please use file names without spaces.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/2024/emma.rachiteanu/img/front view robotic arm.jpg
Comment (alexandruradovici) [reply docs/project/2024/emma.rachiteanu/img/front view robotic arm.jpg:1]: No they don't.

**Review by alexandruradovici — DISMISSED** [review]

**Review by ZamfirAndreiPaul — DISMISSED** [review]

**Comment by alexandruradovici** [comment]
Please fix the build error, most probably you have a broken link.

**Comment by alexandruradovici** [comment]
Please fix the build error, most probably you have a broken link.

**Review by alexandruradovici — APPROVED** [review]

**Review by DTeodor-Alexaandru — APPROVED** [review]

**Pull request merged**